lib/tasks/cleanup.rake

```diff
@@ -0,0 +1,13 @@
+# typed: false
+# frozen_string_literal: true
+
+namespace :cleanup do
+  desc 'Remove unattached files'
+  task uploads: :environment do
+    # Purge all blob keys were created 7 days ago and do not have attachments
+    ActiveStorage::Blob.left_outer_joins(:attachments)
+                       .where('DATE(active_storage_blobs.created_at) = ?', 7.days.ago.to_date)
+                       .where(attachments: { id: nil })
+                       .each(&:purge_later)
```

Comment:
not sure what the expected number of results will look like... guessing it'll be in the tens or hundreds, which is prob fine for this, though if you expected it to be 1000+, you might want to `find_each` instead of `each`, so that it batches things.

Reply:
Done.

Reply:
approved again 🙂